src/openloco/ui.h

```diff
@@ -15,6 +16,12 @@ namespace openloco::ui
         diagonal_arrows,
     };
 
+    struct Resolution
```

Comment:
`resolution_t`?

Reply:
I'm all in favour of what you've proposed in https://github.com/OpenRCT2/OpenLoco/pull/193